lte/gateway/c/core/oai/common/common_types.h

```diff
@@ -278,12 +281,20 @@ typedef struct {
 } ip_address_t;
 
 struct fteid_s;
+
+typedef struct {
+#define MAX_ZONE_CODE_STR_LEN 10
+  uint8_t zone_code[MAX_ZONE_CODE_STR_LEN];
+} regional_subscription_t;
```

Comment:
don't you need the length information for the string here and copy the strlen from the proto message?